perfkitbenchmarker/linux_packages/maven.py

```diff
@@ -0,0 +1,76 @@
+import posixpath
+
+from perfkitbenchmarker import flags
+from perfkitbenchmarker.linux_packages import INSTALL_DIR
+
+try:
+  from urlparse import urlparse
+except ImportError:
+  from urllib.parse import urlparse
```

Comment:
This should be sufficient since we're now 100% python3.  If for some reason it's not (why not?) please use https://six.readthedocs.io/#module-six.moves.urllib.parse

Reply:
@dlott done.